src/plugins/data/public/search/expressions/opensearchaggs.ts

```diff
@@ -296,7 +324,7 @@ export const opensearchaggs = (): OpenSearchaggsExpressionFunctionDefinition =>
       abortSignal: (abortSignal as unknown) as AbortSignal,
     });
 
-    const table: OpenSearchDashboardsDatatable = {
+    const table: OpenSearchDashboardsDatatable & { meta?: any } = {
```

Comment:
Perhaps we can update interface `OpenSearchDashboardsDatatable` to add `meta?` property?